spec/models/feedbook/answer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Feedbook::Answer, type: :model do
+  it { should belong_to(:question) }
+  it { should belong_to(:employees_session) }
+
+  it 'creates a new answer' do
+    question = create(:question)
```

Comment:
Variables inside an example may be better instantiated using `let` or `let!`. For example, for this particular variable you can write:

```
...
let(:question) { create(:question) }

it 'creates a new answer' do
  ...
end
```

Then, you can use `question` inside the example as any other variable. This helps to keep test setup apart from the actual test expectations.